performance_management/views/api.py

```diff
@@ -41,34 +41,50 @@ def get(self, request):
         organizations = Organization.objects.filter(Q(activity__activity_status__firstdate__gte=self.start_date) &
                                                     Q(activity__activity_status__firstdate__lte=self.end_date))
 
-        completed_organizations = Organization.objects.filter(Q(activity__activity_status__firstdate__gte=self.start_date) &
+        completed_organizations =Organization.objects.filter(Q(activity__activity_status__firstdate__gte=self.start_date) &
                                                     Q(activity__activity_status__firstdate__lte=self.end_date) &
-                                                    Q(activity__activity_status__status=self.activity_status))
+                                                    Q(activity__activity_status__status="Completed"))
```

Comment:
@codephillip this is where i need you to review and let me know if this is okay.
you are the guy. Forget the ones below